inspire/dojson/jobs/schemas/jobs-0.0.1.json

```diff
@@ -102,18 +102,46 @@
             "uniqueItems": true
         },
         "url": {
-            "format": "url",
-            "title": "Link to job description",
-            "type": "string"
+            "uniqueItems": true,
+            "items": {
+                "type": "object",
+                "properties": {
+                    "doc_string": {
+                        "type": "string",
+                        "description": "FIXME: What is this!?",
+                        "title": "Doc String"
+                    },
+                    "url": {
+                        "title": "URL",
+                        "type": "string",
+                        "format": "url"
+                    },
+                    "material_type": {
+                        "type": "string",
+                        "title": "Type of material"
+                    },
+                    "description": {
+                        "type": "string",
+                        "title": "Description"
+                    },
+                    "size": {
+                        "type": "integer",
+                        "title": "Size"
+                    }
+                },
+                "title": "URL"
+            },
+            "type": "array",
+            "title": "URLs"
```

Comment:
Maybe not. Currently the base.py under common has the `url` key mapped to this. So that's why jobs and inst was changed to follow the same schema. We could take it out of common and keep it different per "collection".